### ISSUE TITLE
Stats map from scores

### ISSUE THREAD

**Issue by mariliarosa4** [xxlya/BrainGNN_Pytorch#8]
Has anyone been able to generate views from the ROI scores?

**Comment by GreatnessBrain** [comment]
> Has anyone been able to generate views from the ROI scores?

Hi，Do you know what to do now? I am also confused.

**Comment by xxlya** [comment]
In the 130 line of the main.py file:
output, w1, w2, s1, s2 = model(data.x, data.edge_index, data.batch, data.edge_attr, data.pos), s1 can give the scores for all the ROIs, following your defined ROI orders.

**Comment by GreatnessBrain** [comment]
Can you share the corresponding code, how to generate views from the s1 and s2 scores? Many thanks!

**Comment by xdjwolf** [comment]
when I was training, I get the histogram as shown in the figure below.
I used the cc200 template to train, and after topK layer, I got 100 nodes (regions) left. How can I map the existing 100 or 50（two R-pool layers ） points to the 200 points in the original map?
The 100 elements in the S1 array for every person were very close, in the range of 0.48 to 0.64, How can I determine which brain regions are most critical for prediction, do I rank them and get the top 10?


![image](https://user-images.githubusercontent.com/17349541/155886644-c6c4cc1a-2a48-4526-b5d4-f2d807811f24.png)
